business/workloads.go

```diff
@@ -599,26 +587,78 @@ func parseZtunnelLine(line string) *LogEntry {
 		Severity:      "INFO",
 	}
 
+	regex := regexp.MustCompile(models.ZtunnelRegexLine)
+	matches := regex.FindStringSubmatch(line)
+	prop := models.Hbone
+
+	if len(matches) < 18 {
+		// Try additional format
+		regex = regexp.MustCompile(models.ZtunnelInboundRegexLine)
+		matches = regex.FindStringSubmatch(line)
+		prop = models.Ztunnel
+		if len(matches) < 18 {
+			regex = regexp.MustCompile(models.ZtunnelError)
+			matches = regex.FindStringSubmatch(line)
+			prop = models.ErrorLine
+			if len(matches) < 16 {
+				regex = regexp.MustCompile(models.ZtunnelSimpleError)
+				matches = regex.FindStringSubmatch(line)
+				prop = models.SimpleError
+				if len(matches) < 4 {
```

Comment:
I wonder if you need 4 different regexs? I don't have much experience with writing regexps but it seems like you could have a capture group for each prop that is optional in case it doesn't exist?